src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -183,7 +183,7 @@ public sealed class DiagnosticLogManager : AgentService, IDiagnosticLogManager
             }
 
             bool dumpPackagesVerificationResult = AgentKnobs.DumpPackagesVerificationResult.GetValue(executionContext).AsBoolean();
-            if (dumpPackagesVerificationResult && PlatformUtil.RunningOnLinux && !PlatformUtil.RunningOnRHEL6)
+            if (dumpPackagesVerificationResult && PlatformUtil.RunningOnLinux && !PlatformUtil.RunningOnRHELVersion("6"))
```

Comment:
>("6") [](http://example.com/codeflow?start=115&length=5)

should be "6." to match expected logic; I'd suggest making a static field or encapsulating (E.g. exposing RunningOnRHELVersion6 / RunningOnRHELVersion7)

Reply:
Updated. I added back static field RunningOnRHEL6 and RunningOnRHEL7.